cropduster/utils/image.py

```diff
@@ -145,9 +150,18 @@ def process_image(im, save_filename=None, callback=lambda i: i, nq=0, save_param
             save_params.setdefault('quality', get_jpeg_quality(new_images[0].size[0], new_images[0].size[1]))
         if im.format in ('JPEG', 'PNG') and JPEG_SAVE_ICC_SUPPORTED:
             save_params.setdefault('icc_profile', im.info.get('icc_profile'))
-        new_images[0].save(save_filename, **save_params)
-
-        return PIL.Image.open(save_filename)
+        img = new_images[0]
```

Comment:
I'm not 100% consistent in it, but it can be tricky to distinguish between variables named `img` and `image`. Where I thought to, I would either name pil images `im` (which is, oddly, a convention in the PIL documentation), or else used `pil_image`.

Alternatively, if this set of operations were put into a utility function, it might read a lot easier.

Reply:
Yeah, I originally had `from_s3` and `to_s3` utility functions, but reversed..for reasons. I'll revisit and probably rename them to something ~like `open_image` and `write_image`.